Створив секцію Contacts

DonKanElion opened the pull request:
- Розмітка
- Стилізація (моб. версія)
- Форма (моб. версія)
- Фонові зображення (моб. версія)

DonKanElion committed:
Створив секцію Contacts

- Розмітка
- Стилізація (моб. версія)
- Форма (моб. версія)
- Фонові зображення (моб. версія)

DonKanElion committed:
Допрацював Tab, Desk  версіі

Готово все по макету.

kognitiive commented on src/sass/_contacts.scss:
```diff
@@ -0,0 +1,172 @@
+}
+
+.form{
+    display: flex;
```
Тут display: flex; не потрібен

kognitiive commented on src/sass/_contacts.scss:
```diff
@@ -0,0 +1,172 @@
+.form{
+    display: flex;
+    align-items: center;
+    flex-direction: column;
```
flex-direction: column; також прибрати

kognitiive commented on src/sass/_contacts.scss:
```diff
@@ -0,0 +1,172 @@
+
+.form{
+    display: flex;
+    align-items: center;
```
Замість цієї властивості фіксована ширина форми і margin 0 auto

kognitiive reviewed (commented):
У формі прибрати flex

DonKanElion committed:
Вніс правки

- Підігнаш розмір шрифта під макет
- + cursor: pointer на btn
- Марджин на label, через додатковий клас
- Допрацював кнопку.

kognitiive committed:
Merge branch 'main' into Contacts
